README.md

```diff
@@ -2,73 +2,126 @@
 
 [![Build Status](https://travis-ci.org/databricks/spark-sql-perf.svg)](https://travis-ci.org/databricks/spark-sql-perf)
 
-This is a performance testing framework for [Spark SQL](https://spark.apache.org/sql/) in [Apache Spark](https://spark.apache.org/) 1.6+.
+This is a performance testing framework for [Spark SQL](https://spark.apache.org/sql/) in [Apache Spark](https://spark.apache.org/) 1.6+. The toolkit includes the TPC-DS benchmark and a few other sample benchmarks.
 
-**Note: This README is still under development. Please also check our source code for more information.**
+# Build
 
-# Quick Start
+Suppose the spark-sql-perf tool is located at ~/spark-sql-perf. The (non-intuitive) command to build the benchmark is:
 
 ```
-$ bin/run --help
+cd ~/spark-sql-perf
+bin/run --help
+```
+
+The first time you run the command, it may take a while because it may download and install dependencies such as sbt. Future runs should be faster.
+
+# Run a Benchmark
+
+Have a look at src/main/scala/com/databricks/spark/sql/perf/DatasetPerformance.scala. It defines a class `DatasetPerformance` which derives from Benchmark. To run this particular benchmark:
+
+```
+cd ~/spark-sql-perf
+bin/run --benchmark DatasetPerformance
+```
+
+# The TPC-DS Benchmark
+
+## Build dsdgen
+
+This step builds a data generation tool that will be called by `tables.genData()` in a later step.
+Download TPC-DS Tools from http://tpc.org.
+Another option is to use Databricks' fork of dsdgen [here](https://github.com/davies/tpcds-kit).
```

Comment:
It is now https://github.com/databricks/tpcds-kit.
It should be the encouraged default by the README, since it provides the correct implementation for the -filter option that helps generate data on the fly.